Sources/RxComposableArchitecture/Internal/Deprecated.swift

```diff
@@ -7,6 +7,14 @@
 
 import Darwin
 
+// MARK: - Depreacted after 0.50.0:
+@available(
+  *,
+  deprecated,
+  message: "Use 'ReducerBuilder<_, _>' with explicit 'State' and 'Action' generics, instead."
+)
+public typealias ReducerBuilderOf<R: ReducerProtocol> = ReducerBuilder<R.State, R.Action>
```

Comment:
I'll remove this as there is no use in our code base

Reply:
yes, thank you Jeff 🎉